storage/clicks_test.go

```diff
@@ -0,0 +1,75 @@
+package storage
+
+import (
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/mailbadger/app/entities"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClicks(t *testing.T) {
+	db := openTestDb()
+	defer func() {
+		err := db.Close()
+		if err != nil {
+			logrus.Error(err)
+		}
+	}()
+
+	store := From(db)
+	now := time.Now().UTC()
+
+	// Test insert click
+	clicks := []entities.Click{
+		{
+			ID:         1,
+			UserID:     1,
+			CampaignID: 1,
+			Recipient:  "jhon@doe.com",
+			UserAgent:  "android",
+			IPAddress:  "1.1.1.1",
+			CreatedAt:  now,
+			Link:       "s",
+		},
+		{
+			ID:         2,
+			UserID:     1,
+			CampaignID: 1,
+			Recipient:  "jhon@email.com",
+			Link:       "a",
+			UserAgent:  "windows",
+			IPAddress:  "1.1.1.1",
+			CreatedAt:  now,
+		},
+		{
+			ID:         3,
+			UserID:     1,
+			CampaignID: 1,
+			Recipient:  "jhon@email.com",
+			Link:       "a",
+			UserAgent:  "windows",
+			IPAddress:  "1.1.1.1",
+			CreatedAt:  now,
+		},
+	}
+	// insert click 1
+	err := store.CreateClick(&clicks[0])
```

Comment:
@Dzalevski you could also iterate and insert instead of manually adding each one.